client/src/pages/dashboard/users/index.js

```diff
@@ -49,13 +46,12 @@ class Users extends Component {
     isLoading: false,
   };
 
-  editUserInit = (userId) => {
-    const user = this.state.users.filter(user => user._id === userId)[0];
+  editUserInit = userId => {
+    const [user] = this.state.users.filter(({ _id }) => _id === userId);
```

Comment:
Сорян, тупанул, но  есть специальный [метод](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/find) для подобного
```suggestion
    const user = this.state.users.find(({ _id }) => _id === userId);
```